README.md

```diff
@@ -1,46 +1,163 @@
 # Catalyst
 
+The entire bundle of application needed for booting up livepeer's
+decentralised media server in place is termed as [catalyst].
+
 ## Getting Started
 
-go1.17 is recommended because building with < 1.17 (specifically 1.16) can result in issue on arm64 architectures.
+Make sure you have the following installed on the machine before
+proceeding:
 
-From there:
+  - go1.17± or newer
+  - ffmpeg v5 (or newer)
```

Comment:
This is useful, but not necessary — this repo is entirely capable of building the specific ffmpeg version we use in go-livepeer. Which I'd prefer people use, honestly, because it limits the surface area of problems from supporting a ton of arbitrary ffmpeg versions

Reply:
For the minimal usage, where we only run `make download`; ffmpeg won't be compiled. I also think ffmpeg is not much of a requirement (otherwise obs-studio would also be considered a requirement)